CatalogReaderExample/NuGet.Protocol.Catalog/CatalogProcessor.cs

```diff
@@ -0,0 +1,207 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using NuGet.Protocol.Core.Types;
+
+namespace NuGet.Protocol.Catalog
+{
+    public class CatalogProcessor
```

Comment:
Please add documentation to this type and `ProcessAsync`

Reply:
Fixed.